hackathon_site/hardware/api_filters.py

```diff
@@ -0,0 +1,25 @@
+from django_filters import rest_framework as filters
+from django_filters.fields import Lookup
+
+from hardware.models import Hardware
+from hardware.serializers import HardwareSerializer
+
+
+class HardwareFilter(filters.FilterSet):
+    queryset = Hardware
+    serializer_class = HardwareSerializer
+
+    class Meta:
+        model = Hardware
+        fields = ["name", "quantity_available"]
+
+    in_stock = filters.BooleanFilter(label="In stock?", method="filter_in_stock")
```

Comment:
Minor, can you add a `help_text` to these as well, that can just be the same as the label? Turns out it's using the label in the DRF browsable API, but is looking for the help text in swagger